frontend/src/data/providers/GunViolenceYouthProvider.test.ts

```diff
@@ -0,0 +1,83 @@
+import { RACE } from '../utils/Constants'
+import { appendFipsIfNeeded } from '../utils/datasetutils'
+import {
+  autoInitGlobals,
+  getDataFetcher,
+  resetCacheDebug,
+} from '../../utils/globals'
+import { Breakdowns, DemographicType, TimeView } from '../query/Breakdowns'
+import { DatasetId, DatasetMetadataMap } from '../config/DatasetMetadata'
+import { DataTypeId } from '../config/MetricConfig'
+import { Fips } from '../utils/Fips'
+import { MetricId } from '../config/MetricConfig'
+import { MetricQuery, MetricQueryResponse } from '../query/MetricQuery'
+import FakeDataFetcher from '../../testing/FakeDataFetcher'
+import GunViolenceYouthProvider from './GunViolenceYouthProvider'
+
+export async function ensureCorrectDatasetsDownloaded(
+    gunViolenceDatasetId: DatasetId,
+    baseBreakdown: Breakdowns,
+    demographicType: DemographicType,
+    dataTypeId?: DataTypeId,
+    timeView?: TimeView,
+    metricIds?: MetricId[]
+  ) {
+    // If these aren't sent as args, default to []
+    metricIds = metricIds || []
+  
+    const gunViolenceYouthProvider = new GunViolenceYouthProvider()
+    const specificDatasetId = appendFipsIfNeeded(
```

Comment:
we don't need append FIPS here since there aren't any county level data sets

Reply:
oh just saw this is the test file. don't worry you can leave this in doesn't matter. probably better to ensure the tests pass :)